pycds/util.py

```diff
@@ -264,3 +264,32 @@ def create_test_data_from_reflection(read_engine, write_engine):
             wSession.add(new_hist)
 
     wSession.commit()
+
+
+def generic_sesh(sesh, sa_classes, sa_objects):
```

Comment:
Can you maybe provide more comments about what sa_classes and sa_objects are supposed to be? The meaning of the "sa" prefix is not obvious to me...

I do like the way that this enables the developer to reuse the session setup/teardown in the test suite, but I wonder whether you might want to have a look at `pytests` built in parameterization methods, both [for tests](http://doc.pytest.org/en/latest/parametrize.html) and [for test fixuters](http://doc.pytest.org/en/latest/fixture.html#fixture-parametrize). The advantages to using these is that you get a separately run test case for each parameter, which gives you finer granularity when one part (or one instance) of the parameter set fails.